collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -368,40 +389,52 @@ public void recordBean(
         String help = attrDescription + " (" + beanName + attrName + ")";
         String attrNameSnakeCase = toSnakeAndLowerCase(attrName);
 
+        MatchedRule matchedRule = MatchedRule.unmatched();
+
         for (Rule rule : config.rules) {
+          String cacheKey = beanName + attrName;
+          String matchName = beanName + (rule.attrNameSnakeCase ? attrNameSnakeCase : attrName) + ": " + beanValue;
```

Comment:
If we're using the cache, can we put in a dummy value such as cacheEnabled for the bean value so things will obviously break for anyone who tries to extract something from it?